src/signals/incident/components/IncidentPreview/index.test.js

```diff
@@ -44,45 +50,43 @@ describe('<IncidentPreview />', () => {
   describe('rendering', () => {
     it('expect to render correctly', () => {
       isVisible.mockImplementation(() => true);
-      const wrapper = shallow(
-        <IncidentPreview {...props} />
+      const { queryByText } = render(
+        withAppContext(<IncidentPreview {...props} />)
       );
 
-      expect(wrapper).toMatchSnapshot();
+      expect(queryByText('Uw (mobiele) telefoon')).toBeInTheDocument();
```

Comment:
Can you reference the values that are in the `props` var? At least rules out ambiguity.

Reply:
gedaan

Reply:
I don't see any changes. Forgot to push?